modules/upgrade/pages/rolling-upgrade.adoc

```diff
@@ -21,9 +21,13 @@ include::partial$rolling-upgrades/upgrade-limitations.adoc[]
 
 === Review incompatible changes
 
-Starting in version 24.2, xref:manage:cluster-maintenance/manage-throughput.adoc#client-throughput-limits[client throughput limits] are compatible with the https://cwiki.apache.org/confluence/display/KAFKA/KIP-546%3A+Add+Client+Quota+APIs+to+the+Admin+Client[AlterClientQuotas and DescribeClientQuotas^] Kafka APIs. Redpanda determines client throughput limits on a per-broker basis. In earlier versions, client throughput quotas were applied from cluster configurations on a per-shard basis. 
+* Starting in version 24.2, Redpanda removes topic-level configuration overrides for unspecified fields, instead of setting them to their type's default value. This aligns more closely with the AlterConfigs API. 
```

Comment:
@micheleRP THis is inaccurate.  We still have 'topic-level configuration overrides for unspecified fields' in RPK cluster config set, Ansible, Kubernetes, Console, etc. etc. as they use a different API. We are certainly not removing this capability from Redpanda

We are ONLY changing the behavior of the AlterConfigs KAFKA API  to more closely align with the behavior in Apache Kafka.